ephios/settings.py

```diff
@@ -316,6 +320,19 @@ def GET_SITE_URL():
     return site_url
 
 
+def GET_USERCONTENT_URL():
+    return MEDIA_URL
+
+
+def GET_USERCONTENT_QUOTA():
+    try:
+        disk_usage = os.statvfs(MEDIA_ROOT)
+        return disk_usage.f_bavail * disk_usage.f_frsize
+    except AttributeError:
+        # not available on windows
+        return None
```

Comment:
mega upload goes brrrrrrrrrrrrrrrr

Reply:
do you have a better idea on what to do here? but only relevant in dev anyway

Reply:
Just add some constant like 1GB. Or even better, let's use an environment variable. Non-negative values mean available disk space minus that value and positive is the limit itself or smth like that

Reply:
now works on windows as well